infra/cifuzz/build_fuzzers_entrypoint.py

```diff
@@ -28,32 +28,6 @@
     level=logging.DEBUG)
 
 
-def get_pr_ref(event_path):
-  """Returns the PR ref from |event_path|."""
-  with open(event_path, encoding='utf-8') as file_handle:
-    event = json.load(file_handle)
-    return 'refs/pull/{0}/merge'.format(event['pull_request']['number'])
```

Comment:
I'm not sure it's safe to replace `pull_request.number` with `GITHUB_REF`. Judging by https://github.com/google/oss-fuzz/pull/3845, at the time `GITHUB_REF` wasn't set correctly when jobs were triggered manually. Did GitHub fix that bug?

Reply:
Thanks for pointing this out. I'll change it back.